src/runAll.js

```diff
@@ -32,14 +31,16 @@ module.exports = async function runAll(config) {
 
   debug('Resolved git directory to be `%s`', gitDir)
 
-  sgf.cwd = gitDir
-
   /* files is an Object{ filename: String, status: String } */
-  const files = await pify(sgf)('ACM')
```

Comment:
What about those filters in the new implementation? We definitely don’t want deleted or moved files there. Let’s add a test?

Reply:
Good catch! I’m on vacation this week, so let’s get back to these later.

Reply:
added `--diff-filter=ACM`.